lingua_franca/res/text/en-us/normalize.json

```diff
@@ -99,7 +99,78 @@
     "you'd've": "you would have",
     "you'll": "you will",
     "you're": "you are",
-    "you've": "you have"
+    "you've": "you have",
+    "I'm'a": "I am about to",
```

Comment:
I think this is more commonly "I am going to," the result of the word "gonna" slowly being swallowed, but maybe a linguist would disagree.

I'ma go get some lunch

I'ma do it tomorrow